m4opt/synphot/background/_cerenkov/__init__.py

```diff
@@ -0,0 +1,370 @@
+
+    References
+    ----------
+    This module is a Python adaptation of the MATLAB `Cerenkov` function from the MAATv2 package:
+    https://www.mathworks.com/matlabcentral/fileexchange/128984-astropack-maatv2
+
+    It incorporates:
+    - The AE8 trapped electron flux model from radiation belt :doc:`NASA AE8/AP8, IRBEM <irbem:api/radiation_models>`,
+    constraining the flux of charged particles in Earth's radiation belts.
+    - Stopping power and refractive index models :footcite:`2018PASP..130g5002S, 2019PASP..131e4504O`.
+    - The conservative noise modeling approach from ULTRASAT :footcite:`2024ApJ...964...74S`.
+    - The energy grid used for interpolation was read from Figure 6 :footcite:`2016PASP..128c5005K`.
+
+    .. footbibliography::
+
+    Examples
+    --------
+    Cenrenkov background spectrum for a given observer location and time.
+
+    >>> from astropy.coordinates import EarthLocation
+    >>> from astropy.time import Time
+    >>> from astropy import units as u
+    >>> from m4opt.synphot.background._cerenkov import CerenkovBackground
+    >>> observer_location = EarthLocation.from_geodetic(lon=15 * u.deg, lat=0 * u.deg, height=35786 * u.km)
+    >>> obstime = Time("2025-05-18T02:48:00Z")
+    >>> background = CerenkovBackground(particle='e', solar='max', material='SiO2_suprasil_2a')
+    >>> spectrum = background.cerenkov_emission(observer_location, obstime)
+    >>> wavelength = spectrum.waveset
+    >>> spectrum(wavelength[0])
+    <Quantity 5.5374254e-07 PHOTLAM>
+
+    .. plot::
+        :include-source: False
+        :caption: Cerenkov background spectrum for GEO at 2025-05-18T02:48:00Z
+
+        from astropy.coordinates import EarthLocation, ICRS
+        from astropy_healpix import HEALPix
+        from astropy.time import Time
+        from astropy import units as u
+        import numpy as np
+        import matplotlib.pyplot as plt
+        plt.rcParams["font.family"] = "Times New Roman"
+
+        from m4opt.synphot.background._cerenkov import CerenkovBackground
+        from m4opt.synphot import observing
+
+        observer_location = EarthLocation.from_geodetic(
+            lon=15 * u.deg, lat=0 * u.deg, height=35786 * u.km
+        )
+        obstime = Time("2025-05-18T02:48:00Z")
+        hpx = HEALPix(nside=512, frame=ICRS())
+        coord = hpx.healpix_to_skycoord(np.arange(hpx.npix))
+
+        with observing(observer_location=observer_location, target_coord=coord, obstime=obstime):
+            cerenkov_model = CerenkovBackground(particle='e', solar='max', material='SiO2_suprasil_2a')
+            spectrum = cerenkov_model.cerenkov_emission(observer_location, obstime)
+            wavelength = spectrum.waveset
+            intensity = spectrum(wavelength)
+
+        fig, ax = plt.subplots(figsize=(7, 5))
+        ax.plot(wavelength, intensity)
+        ax.set_xlabel(rf"Wavelength [{wavelength.unit}]")
+        ax.set_ylabel(rf"Intensity [{intensity.unit} (erg / s cm$^{{2}}$ Hz)]")
```

Comment:
This is probably redundant because `{intensity.unit}` probably _renders to_ `erg / s cm$^{{2}}$ Hz)`.